python/ECCEmailer.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
+
+#
+# Send an automated email based on provided information.
+#
+# This script presumes that smtp-relay.gmail.com has been setup to
+# accept emails from this public IP address without authentication.
+# To set this up:
+#
+# 1. Login to admin.google.com as an administrator.
+# 2. Apps
+# 3. G Suite
+# 4. Gmail
+# 5. Advanced settings (at the bottom of the list)
+# 6. Scroll down to Routing and find SMTP relay service
+# 7. Add the desired IP address:
+#    - require TLS encryption: yes
+#    - require SMTP authentication: no
+#    - allowed senders: Only addresses in my domains
+#
+# And then make sure that the sender is actually a valid email address
+# in the ECC Google domain.
```

Comment:
You are sending from no-reply@epiphanycatholicchurch.org, which is not a valid email address.  Is this comment wrong?

Reply:
@cabralhj Github pro tip: when you fix something that was commented on in the pull request, you should click the "Resolve" button on that item so that the reviewer knows that it was addressed.

Reply:
This comment still stands: is the comment wrong?